src/Microsoft.Azure.WebJobs.Storage/Blob/IStorageBlobProperties.cs

```diff
@@ -37,5 +37,10 @@ internal interface IStorageBlobProperties
         /// Gets the LeaseStatus of the blob.
         /// </summary>
         LeaseStatus LeaseStatus { get; }
+
+        /// <summary>
+        /// Gets the underlying SDK object.
+        /// </summary>
+        BlobProperties SdkObject { get; }
```

Comment:
Yeah, I'm following the existing naming pattern for these mock types. SdkObject is used all over.